grpo-finetuning-qwen3/agent-skill/grpo-finetune/run_pipeline.py

```
@@ -0,0 +1,192 @@
+    banner("RESULTS")
+    print(f"  accuracy   {finetuned:.1%}", flush=True)
+    print(f"  model      {deployment}", flush=True)
+    print(f"  elapsed    {time.time() - t0:.0f}s", flush=True)
+
+    # 4. sample inference
+    invoices_path = Path(args.invoices)
+    if invoices_path.exists():
+        banner("STEP 4 · sample inference")
+        print("Running the fine-tuned model on sample invoices...\n", flush=True)
+        time.sleep(1)
+        subprocess.run([sys.executable,
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)